Configuration.cs

```diff
@@ -10,6 +10,9 @@ public sealed class Configuration : IPluginConfiguration
 
     #region Saved configuration values
 
+    public TitleGroup randomTitleGroup { get; set; }
```

Comment:
It would -in theory- make more sense to use the name of the group here to reduce the chance your synchronization breaks (e.g. different characters not having the same titles unlocked); in practice I'd prefer just using  `TitleGroups.Where(x => x.IsDefault)` to find whichever group would be picked.